packages/dds/map/src/test/mocha/directory.spec.ts

```diff
@@ -34,7 +34,6 @@ function createLocalMap(id: string) {
     return factory.create(new MockFluidDataStoreRuntime(), id);
```

Comment:
doesn't need to be in this PR, but i think we need a fuzz/stress test for this dds, there are examples in merge-tree (packages\dds\merge-tree\src\test\client.conflictFarm.spec.ts) and in sequence (packages\dds\sequence\src\test\intervalCollection.fuzz.spec.ts)

these types are test are really good at testing eventual consistency, and ensuring multiple clients converge to the same state.

Reply:
open an issue for someone to build this in the near-ish future

Reply:
Opened a bug in ADO: https://dev.azure.com/fluidframework/internal/_workitems/edit/1865 @anthony-murphy